config/sara/README.md

```diff
@@ -1,38 +1,53 @@
+
+A `symptom` will be identified by its name, which is the key in the symptom object, a `value`, `type`, `required`, `group` and `threshold_operator`.
+
 ### value
-The `value` of a symptom defines the threshold of the symptom, this is the value that is considered as
-symptomatic for the symptom which it is defining, only symptoms marked as 'required' will be considered when
-determining whether or not a patient is symptomatic. For float and integer symptoms, a reported symptom
-will be compared using the specified `threshold_operator` to the specified `value` and will be considered as symptomatic. if the result of the comparison is true.
+
+The value of a symptom defines the threshold value that is considered symptomatic for the symptom it defines; only symptoms marked as 'required' are considered when determining whether or not a patient is symptomatic. For float and integer symptoms, a reported symptom will be compared using the specified threshold_operator to the specified value and will be considered as symptomatic if the result of the comparison is true. For boolean symptoms, a reported symptom will be considered as symptomatic if the threshold and reported values are both true.
+
 ### threshold_operator
+
 Available values for the `threshold_operator` field are `Less Than`, `Less Than Or Equal`,
 `Greater Than`, `Greater Than Or Equal`, `Equal` and `Not Equal`.
+
 ### type
+
 Available values for the `type` field in a symptom are `FloatSymptom`, `IntegerSymptom`, or `BoolSymptom`.
+
 ### group
+
 The group parameter specifies how many symptoms within the same group must pass their threshold values in order to be considered symptomatic. This enables the ability to construct assessments that will be considered symptomatic in the case where N or more of a group of symptoms pass their threshold values. eg: Cough, Fever or any two of Difficulty Breathing, Headache or Vomit. This example would be acheived by setting the Cough and Fever symptoms to be in group 1, and the Difficulty Breathing, Headache and Vomit symptoms to be in group 2. When a group is not specified the default value is 1.
 Note: Sub-jurisdictinos can add symptoms to groups, but should not be specifying their own groups.
+
 ### required
-The `required` field can either take the value `true` of `false`. Only symptoms with `required` set to `true` will be taken into consideration during symptomatic assessment calculation. Additionally, symptoms that have the `required` attribute set to `false` will be omitted from the voice and SMS assessment prompts.
+
+The `required` field can either take the value `true` or `false`. Only symptoms with `required` set to `true` will be taken into consideration during symptomatic assessment calculation. Additionally, symptoms that have the `required` attribute set to `false` will be omitted from the voice and SMS assessment prompts.
 
 ## Translations and Notes
```

Comment:
Since this still pertains to the Symptoms
```suggestion
### Translations and Notes
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1254/commits/09f765ebf231a8cfc1897e10b648ac8eb5f27be0